commons/src/test/java/io/hyscale/commons/utils/TestHyscaleFilesUtil.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.utils;
+
+import io.hyscale.commons.constants.ToolConstants;
+import io.hyscale.commons.exception.HyscaleException;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.net.URL;
+
+public class TestHyscaleFilesUtil {
+    private static URL SAMPLE_FILE_URL;
+    private static String SAMPLE_FILE;
+    private static File sampleFile;
+    private static String testDirPath;
+    private static String testFilePath;
+    private static File testDir;
+    private static String sampleFileName = "sample.txt";
+
```

Comment:
1. Why are some variables with Uppercase and others with Lowercase
2.Why sampleFileName is initialized globally and rest are initialised in before each annotation ?